scss/backend/dot-admin/portlets/_view-workflow.scss

```diff
@@ -17,8 +17,101 @@
 }
 
 .view-actions__who-can-use {
+    width: 80%;
+
     .who-can-use__list {
         min-width:230px;
     }
-    
 }
+
+.portlet-wrapper .view-actions {
+    height: 800px;
+    padding-left: $basic-padding*3;
+
+    .view-actions__back-btn {
+        height: 16px;
+        position: absolute;
+        padding: 0px;
+        left: -12px;
+        top: 10px;
+
+        .dijitButtonNode {
+            padding: 0 8px;
+        }
+
+        i {
+            transform: rotate(-90deg);
+        }
+    }
+
+    .container-fluid, .container-fluid > .row:last-child, > form {
+        height: 100%;
+    }
+
+    .view-actions__permissions {
+        border-left: 1px solid $gray-ligther;
+        height: 100%;
+        position: relative;
+        padding-left: $basic-padding*3;
+    }
+
+    .view-actions__sub-actions {
```

Comment:
There is already styles for this, class: ```view-actions__edit-actions``` (I know, name could be more general) :)